wallbox_modbus_registers.yaml

```diff
@@ -65,7 +65,7 @@ charging_state: &charging 1
 # Connected and waiting for car demand; sometimes shortly goes to this status when action = start
 waiting_state: &waiting 2
 
-# Connected and waiting for next schedule; this occurs when a charging session is scheduled via the Wallbox app.
+# Connected and waiting for next schedule; this occurs when a charging is scheduled via the Wallbox app.
```

Comment:
Accidental deletion? Otherwise: ~a~ charging is

Reply:
Copy error